cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When("the user clicks the Add concepts button", () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then("the user should be on the Import existing concept page", () => {
+  cy.visit("/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/openmrs/collections/")
+});
+
+Given("the user is on the Import existing concept page", () => {
+  cy.visit("/orgs/CIEL/sources/CIEL/concepts/?addToDictionary")
```

Comment:
We should fill in the `addToDictionary` here, e.g.,

```suggestion
  cy.visit(`/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`)
```